Source/CesiumRuntime/Private/CesiumGltfComponent.cpp

```diff
@@ -3849,13 +3859,8 @@ BuildChaosTriangleMeshes(
     int32 vIndex1 = indices[index0];
     int32 vIndex2 = indices[index0 + 2];
 
-    if (!isTriangleDegenerate(
-            vertices.X(vIndex0),
-            vertices.X(vIndex1),
-            vertices.X(vIndex2))) {
-      triangles.Add(Chaos::TVector<int32, 3>(vIndex0, vIndex1, vIndex2));
-      faceRemap.Add(i);
-    }
+    triangles.Add(Chaos::TVector<int32, 3>(vIndex0, vIndex1, vIndex2));
+    faceRemap.Add(i);
```

Comment:
Noticed you removed the call to `isTriangleDegenerate`, and rightly so. It looks unused now, and might want to consider removing that function completely so people don't go down this rabbit hole again.